src/irmin/tree.ml

```diff
@@ -430,12 +430,8 @@ module Make (P : Backend.S) = struct
       | Pruned of hash
 
     and t = { mutable v : v; info : info }
-    (** [t.v] has 3 possible states:
-
-        - A [Map], only after a [Tree.of_concrete] operation.
-        - A [Value], only after an add, a remove, temporarily during an export
-          or at the end of a merge.
-        - It is otherwise a [Key].
+    (** For disussion of [t.v]'s states, see interface documentation for
+        [Tree.inspect].
```

Comment:
Suggested change: `{!Tree_intf.inspect}` (or similar) i.e., directly link to the inspect function in *Tree_intf*

Reply:
:+1: good idea!